app/public/js/home.js

```diff
@@ -60,11 +60,53 @@ function initMap()
 		google.maps.event.trigger(map, "resize");
```

Comment:
Did you reload the page to see if the marker stays in the clicked position?

Steps to test:
1. Click the middle of a lake.
1. Reload page.
1. Check if the marker is still in the same part of the lake.

I didn't test it but suspect that something will break in the above scenario because you're not interpretting your coordinate format anywhere in your changes.  You set the coordinates in the address input but there's no code to extract those values and use them to place the marker.

Reply:
I didn't change the marker position for the locations that doesn't map to any address.

Reply:
Do you have a reason why you think it is fine to not move the map or marker when no nearby address can be found?

Reply:
First i thought that setting a marker to those weird locations would not be of any use.

Reply:
Why do you think the coordinate expressions are being set when the address is unavailable?

Why do you think the address input is available on the home page and how is that value used?  Did you change it and see how the search results look on the location search's map?

![image](https://user-images.githubusercontent.com/11838191/33100643-1968f6fa-cee3-11e7-884d-0a0b32a7c4ec.png)

Here's a link to hit after you put an address into http://app.accesslocator.com
http://app.accesslocator.com/location-search?keywords=&order_by=name&location_tag_id=1&view=map

Reply:
-> I think the coordinate expressions are only shown to the user when there is no nearby locations to the clicked point.
-> Address input value is being used to find the nearby search results and if i change it search results are changed accordingly.
->i have already checked the map on the search results page. Location which is shown by the blue marker is the mapped address( Address input provided on the home page ) not exactly the point we click on the map.

Reply:
I can see how it behaves on my computer as I'm testing it.

It works fairly well with the addresses.
When the coordinates are used, I want the marker and map to move to it.

It may be an edge case but I want it to work in a consistent way.  If someone is fishing in a big lake and looking for a nearby place to get food, I want him to be able to click where he is in the lake and still search for places based on the clicked/touched location.  I don't want the app to essentially ignore what he did and just list the coordinates in the home page as if his request hit a brick wall.

Reply:
Sure i'll modify it to move the marker on coordinates address as well.